### PR TITLE
Entities & Glance Card: Add state color to editors

### LINKED ISSUE #6621: Toggle for "state_color" under entities card
## The request
Please add a toggle for state_color for entity card. Currently if i add the option state_color: i lose the ability to use visual editor. There is plenty of space next to "Show header Toggle" to put "State Color Toggle". 
chrome_u84R7dHslN
chrome_n9uP4iAlHA_LI

## The alternatives
I am fine with not having the toggle but I don't want the ability to use Visual editor to be disabled just because i want state color on. Having to delete that line every time i want to use visual editor is not a great experience either. Besides i think many people would fine state color change to be very useful for many switches.

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

image

image

## Type of change

- [x] New feature (thank you!)

## Additional information

- This PR fixes or closes issue: fixes #6621
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -2164,6 +2164,7 @@
             "entities": {
               "name": "Entities",
               "show_header_toggle": "Show Header Toggle?",
+              "state_color": "Show State Color?",
```
Comment (zsarnett) [inline src/translations/en.json:2167]: This as the field label feels weird but at the same time not. I just could think of another wording.

Comment (KTibow) [reply src/translations/en.json:2167]: `Change Color Automatically?` `Show Auto Color?`

Comment (Misiu) [reply src/translations/en.json:2167]: Maybe "Use State Color" or "Apply State Color"?
Naming is hard...

Comment (bramkragten) [reply src/translations/en.json:2167]: Color icons based on state

Comment (zsarnett) [reply src/translations/en.json:2167]:
```suggestion
              "state_color": "Color Icons based on state",
```

File: src/translations/en.json [hunk 2]
```diff
@@ -2192,6 +2193,7 @@
             "glance": {
               "name": "Glance",
               "columns": "Columns",
+              "state_color": "Show State Color?",
```
Comment (zsarnett) [inline src/translations/en.json:2196]:
```suggestion
              "state_color": "Color icons based on state",
```

File: src/translations/en.json [hunk 3]
```diff
@@ -2193,7 +2193,7 @@
             "glance": {
               "name": "Glance",
               "columns": "Columns",
-              "state_color": "Show State Color?",
+              "state_color": "Color icons based on state",
```
Comment (Misiu) [inline src/translations/en.json:2196]: all other options have question marks on end ( for example https://github.com/home-assistant/frontend/blob/dev/src/translations/en.json#L2249)
shouldn't it be added here for consistency?

Comment (zsarnett) [reply src/translations/en.json:2196]:
```suggestion
              "state_color": "Color icons based on state?",
```

File: src/translations/en.json [hunk 4]
```diff
@@ -2164,6 +2164,7 @@
             "entities": {
               "name": "Entities",
               "show_header_toggle": "Show Header Toggle?",
+              "state_color": "Color icons based on state",
```
Comment (zsarnett) [inline src/translations/en.json:2167]:
```suggestion
              "state_color": "Color icons based on state?",
```

File: src/panels/lovelace/editor/config-elements/hui-glance-card-editor.ts [hunk 5]
```diff
@@ -89,6 +90,10 @@ export class HuiGlanceCardEditor extends LitElement
     return this._config!.show_state || true;
   }
 
+  get _state_color(): boolean {
+    return Boolean(this._config!.state_color);
+  }
```
Comment (bramkragten) [inline src/panels/lovelace/editor/config-elements/hui-glance-card-editor.ts:95]: For glance state color is default true right? So should be:
```suggestion
  get _state_color(): boolean {
    return this._config!.state_color ?? true;
  }
```

File: src/translations/en.json [hunk 6]
```diff
@@ -2192,6 +2193,7 @@
             "glance": {
               "name": "Glance",
               "columns": "Columns",
+              "state_color": "Color icons based on state?",
```
Comment (bramkragten) [inline src/translations/en.json:2196]: Should we move it to generic?

**Review by bramkragten — APPROVED** [review]

**Pull request merged**